From 015e826ff5c0c56233f3731ea199e2b8633a84c8 Mon Sep 17 00:00:00 2001
From: Rebecca Chang Swee Fun <rebecca.chang@starfivetech.com>
Date: Fri, 14 Jan 2022 01:04:19 +0000
Subject: [PATCH 08/25] skia: add riscv64

Signed-off-by: Rebecca Chang Swee Fun <rebecca.chang@starfivetech.com>
---
 skia/BUILD.gn | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/skia/BUILD.gn b/skia/BUILD.gn
index 9e940198dae11..7d79544f8ea7e 100644
--- a/skia/BUILD.gn
+++ b/skia/BUILD.gn
@@ -811,6 +811,8 @@ skia_source_set("skia_opts") {
     sources = skia_opts.none_sources
   } else if (current_cpu == "s390x") {
     sources = skia_opts.none_sources
+  } else if (current_cpu == "riscv64") {
+    sources = skia_opts.none_sources
   } else {
     assert(false, "Need to port cpu specific stuff from skia_library_opts.gyp")
   }
-- 
2.25.1

