From 9df3fb93bedabde635c1bce54332b0a8b050327d Mon Sep 17 00:00:00 2001
From: Rebecca Chang Swee Fun <rebecca.chang@starfivetech.com>
Date: Wed, 19 Jan 2022 03:12:16 +0000
Subject: [PATCH 23/27] build: build_config.h: redefine for riscv

Signed-off-by: Rebecca Chang Swee Fun <rebecca.chang@starfivetech.com>
---
 build/build_config.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/build_config.h b/build/build_config.h
index e567c61d47091..65a795be4aa35 100644
--- a/build/build_config.h
+++ b/build/build_config.h
@@ -325,7 +325,7 @@
 #define ARCH_CPU_64_BITS 1
 #define ARCH_CPU_LITTLE_ENDIAN 1
 #elif defined(__riscv) || (__riscv_xlen == 64)
-#define ARCH_CPU_RISCV64_FAMILY 1
+#define ARCH_CPU_RISCV_FAMILY 1
 #define ARCH_CPU_RISCV64 1
 #define ARCH_CPU_64_BITS 1
 #define ARCH_CPU_LITTLE_ENDIAN 1
-- 
2.25.1

